azure/azure_linux_docker_app_service/monitoring.tf

```
@@ -0,0 +1,126 @@
+locals {
+  healthcheck_endpoint = "https://${azurerm_app_service.current.name}.azurewebsites.net/api/healthcheck"
```

Suggestion: could use [`default_site_hostname`](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/app_service#default_site_hostname) here.
```suggestion
  healthcheck_endpoint = "https://${azurerm_app_service.current.default_site_hostname}/api/healthcheck"
```